web/src/pages/index.page.tsx

```diff
@@ -50,8 +54,30 @@ const Home: NextPage = () => {
 
       <section id="how-it-works">
         <StyledBox>
-          <Typography variant="h4">How it works</Typography>
-          <YoutubeEmbed embedId="yM8RrwQWeJc" />
+          <Box sx={{ height: 600, width: "70%" }}>
+            <Typography variant="h4">How it works</Typography>
+            <Carousel autoPlay={false}>
+              <YoutubeEmbed embedId="yM8RrwQWeJc" />
```

Comment:
Since the `YoutubeEmbed` has a max width smaller than the Carousel's, I think we should horizontally center children of the Carousel. Maybe `flex` with `align-items` on a `styled(Carousel)` would work?
![image](https://github.com/amelioro/ameliorate/assets/13872370/c29e7532-6a44-41fe-9cd3-0b5fa02d84bf)